templates/custom/nodes/node--contact--teaser.html.twig

```diff
@@ -73,7 +73,7 @@
 
 {{ attach_library('classy/node') }}
 
-<div class="teaser teaser-contact">
+<div class="teaser">
   <address {{ attributes.addClass('contact-fiche') }}>
```

Comment:
> The` <address>` tag defines the contact information for the author/owner of a document or an article.

Is this the case?

Reply:
Nope, not the case but this isn't relevant anymore since this content-type will be removed... It's replaced by a custom entity in Drupal. 

The removal of the class is merely a styling-fix for testing.

Reply:
Allright, I'll be temporarily blind.